24life/blocks/article-hero-video/article-hero-video.css

```diff
@@ -89,6 +108,10 @@ div.article-hero-video-wrapper {
   filter: brightness(0.80);
 }
 
+.cards.block {
+  text-align: center;
+}
+
```

Comment:
This will change any card blocks anywhere on the site. Is this really intended?

Reply:
There is a card block with 24hourfitness - that is not in the center. I have removed it & now 24 hour fitness card is not in the center. Pl advise @synox 

![Uploading image.png…]()

Reply:
Cards is not the right component for this. You can use Columns, leave the first and 3rd column empty and add it to the second column.

Reply:
@synox  - I choose cards block owing to similar / same built of the intended image as shown below  - **image at the top and its content at the bottom** and it looks perfectly the same as there on the site - https://www.24life.com/newsletter-full/

<img width="1763" alt="image" src="https://github.com/hlxsites/24life/assets/37831850/ca6dab55-ad4c-44d2-b39f-f4bf85eb08fb">

Indeed we do not want to target any other card block apart from card block just above newsletter-form-container block hence this works for me very well - 

.cards-container:has(+ .newsletter-form-container){
  text-align: center;
}

and its working. 

Fixed